### DISCUSSION TITLE
Daisy UI causes unwanted changes to my app's layout and theme

### DISCUSSION THREAD

**Discussion by franknmungai** [saadeghi/daisyui#2068] [Q&A]
I'm working on a next.js project (v13.4.6) using Tailwindcss. I just installed DaisyUI to my project and added it to my plugins list in the tailwind config file as per the daisyui website documentation. However, it seems that DaisyUI makes changes to my website theme and layout (which I had already designed using Tailwindcss). The changes caused by DaisyUI are quite undesirable. What could be the issue and how can I avoid this?

**Comment by saadeghi** [comment]
> DaisyUI makes changes to my website theme and layout

Can you explain what unwanted changes?  
I need more details.
